samcli/local/lambdafn/utils.py

```diff
@@ -0,0 +1,33 @@
+"""
+Helper methods that aid with in setting up local lambda function.
+"""
+
+import os
+import posixpath
+import re
+try:
+    import pathlib
+except ImportError:
+    import pathlib2 as pathlib
+
+from functools import wraps
+
+
+def path_modifier(func):
+    """
+    This method decorator allows conversions of windows style path to unix style path
```

Comment:
Can you update this to use numpy doc strings instead? We are moving off this format onto the numpy format with new docs.